UnitTests/BraintreeCardTests/BTCardClient_Tests.swift

```diff
@@ -5,15 +5,18 @@ import XCTest
 
 class BTCardClient_Tests: XCTestCase {
     let customerInputValidationErrorKey: String = "BTCustomerInputBraintreeValidationErrorsKey"
+    let sandboxAuthorization: String = "sandbox_9dbg82cq_dcpspy2brwdjr3qn"
+    
     // MARK: - ClientAPI
     
     func testTokenization_postsCardDataToClientAPI() {
         let expectation = self.expectation(description: "Tokenize Card")
 
-        let mockAPIClient = MockAPIClient(authorization: "development_tokenization_key")!
+        var mockAPIClient = MockAPIClient(authorization: "development_tokenization_key")!
```

Comment:
Is there a reason this needs to be a var now? It doesn't look like we are mutating it but I may be missing something.

Reply:
ahh good catch - that's true there's no reason for it to be `var` (think i changed it while testing smth and forgot to revert it back)